1526631193549	Sync.LogManager	DEBUG	Flushing file log
1526631193612	Sync.LogManager	DEBUG	Log cleanup threshold time: 1525767193612
1526631193849	Sync.LogManager	DEBUG	Done deleting files.
1526631194485	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1526631213171	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1526631213171	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1526631213176	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1526631214171	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1526631215154	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1526631215154	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1526631215170	Sync.Service	DEBUG	User-Agent: Firefox/60.0 (Linux x86_64) FxSync/1.62.0.20180509234620.desktop
1526631215170	Sync.Service	INFO	Starting sync at 2018-05-18 15:13:35 in browser session 8IYZaSGUJ7Gm
1526631215170	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1526631215178	Sync.Status	INFO	Resetting Status.
1526631215178	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1526631215183	Sync.Resource	WARN	GET request to https://sync-600-us-west-2.sync.services.mozilla.com/1.5/92728811/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526631215183	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1526631215183	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1526631215184	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526631215185	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1526631215185	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 4799445 ms.
1526631215186	Sync.SyncScheduler	DEBUG	Next sync in 4799445 ms. (why=client-backoff-schedule)
1526631215191	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526631215192	Sync.ErrorHandler	DEBUG	Addons installed: 8
1526631215192	Sync.ErrorHandler	DEBUG	 - Firefox Multi-Account Containers, version 6.0.0, id @testpilot-containers
1526631215192	Sync.ErrorHandler	DEBUG	 - GNOME Shell integration, version 10.1, id chrome-gnome-shell@gnome.org
1526631215192	Sync.ErrorHandler	DEBUG	 - Web Extension for Shaarli, version 2.0.0, id shaarli@imirhil.fr
1526631215193	Sync.ErrorHandler	DEBUG	 - DuckDuckGo Privacy Essentials, version 2018.5.1, id jid1-ZAdIEUB7XOzOJw@jetpack
1526631215193	Sync.ErrorHandler	DEBUG	 - FoxyProxy Standard, version 6.2, id foxyproxy@eric.h.jung
1526631215193	Sync.ErrorHandler	DEBUG	 - Zotero Connector, version 5.0.39, id zotero@chnm.gmu.edu
1526631215193	Sync.ErrorHandler	DEBUG	 - Sync Tab Groups, version 0.6.9, id sync-tab-groups@eric.masseran
1526631215193	Sync.ErrorHandler	DEBUG	 - uBlock Origin, version 1.16.6, id uBlock0@raymondhill.net
